src/Relation/Unary/Properties.agda

```diff
@@ -15,9 +15,10 @@ open import Level using (Level)
 open import Relation.Binary.Core as Binary
 open import Relation.Binary.Definitions
   hiding (Decidable; Universal; Irrelevant; Empty)
-open import Relation.Binary.PropositionalEquality.Core using (refl)
+open import Relation.Binary.PropositionalEquality.Core using (refl; _≗_)
 open import Relation.Unary
-open import Relation.Nullary.Decidable using (yes; no; _⊎-dec_; _×-dec_; ¬?)
+import Relation.Nullary.Decidable as Dec
+open import Relation.Nullary.Decidable as Dec using (yes; no; _⊎-dec_; _×-dec_; ¬?; map′; does)
```

Comment:
Apologies for returning to this after a few days' away.

L20 is now redundant, being superseded/'overridden' by L21, so there are several possible tweaks to eliminate such redundancy:
* if we want to preserve the distinction between L20 and L21, by virtue of the fact that L21 imports from `Decidable.Core` (reexported publicly by `Decidable`), then change L21 to the narrower import
* if we want to preserve L20 in its own right, then we could 'optimise' L21 simply to `open Dec using...`, since that local/qualified module name is after L20 already in scope
* but, to my mind, simplest and best of all, would simply be to delete L20, leaving just
```suggestion
open import Relation.Nullary.Decidable as Dec using (yes; no; _⊎-dec_; _×-dec_; ¬?; map′; does)
```